script/efr32-definitions

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# shellcheck disable=SC2034  # $efr32_platforms is unused in this file, but is
```

Comment:
You could change efr32_platforms so that it is not a variable but a function (say efr32_get_plattforms) that returns the platforms.  This way you don't need this warning disable and is more like the efr32_get_boards.